specification/ai/OpenAI.Assistants/client.tsp

```diff
@@ -156,6 +156,62 @@ namespace Azure.AI.OpenAI.Assistants {
 
   // Ensure some shared types remain public despite interaction with previously internalized details
 
+  // Ensure generator inclusion of streaming "leaf" types not yet referenced by a route
+
+  @@usage(MessageDeltaTextFileCitationAnnotation, Usage.output);
+  @@access(MessageDeltaTextFileCitationAnnotation, Access.public);
+
+  @@usage(MessageDeltaTextFilePathAnnotation, Usage.output);
+  @@access(MessageDeltaTextFilePathAnnotation, Access.public);
+
+  @@usage(MessageDeltaImageFileContent, Usage.output);
+  @@access(MessageDeltaImageFileContent, Access.public);
+
+  @@usage(MessageDeltaTextContent, Usage.output);
+  @@access(MessageDeltaTextContent, Access.public);
+
+  @@usage(RunStepDeltaMessageCreationObject, Usage.output);
+  @@access(RunStepDeltaMessageCreationObject, Access.public);
+
+  @@usage(RunStepDeltaFunction, Usage.output);
+  @@access(RunStepDeltaFunction, Access.public);
+
+  @@usage(RunStepDeltaCodeInterpreterLogOutput, Usage.output);
+  @@access(RunStepDeltaCodeInterpreterLogOutput, Access.public);
+
+  @@usage(RunStepDeltaCodeInterpreterImageOutputObject, Usage.output);
+  @@access(RunStepDeltaCodeInterpreterImageOutputObject, Access.public);
+
+  @@usage(RunStepDeltaChunk, Usage.output);
+  @@access(RunStepDeltaChunk, Access.public);
+
+  @@usage(MessageDeltaChunk, Usage.output);
+  @@access(MessageDeltaChunk, Access.public);
+
+  // // Stream events
+
+  @@usage(AssistantStreamEvent, Usage.output);
+  @@access(AssistantStreamEvent, Access.internal);
+
+  @@usage(ThreadStreamEvent, Usage.output);
+  @@access(ThreadStreamEvent, Access.internal);
+
+  @@usage(RunStreamEvent, Usage.output);
+  @@access(RunStreamEvent, Access.internal);
+
+  @@usage(RunStepStreamEvent, Usage.output);
+  @@access(RunStepStreamEvent, Access.internal);
+
+  @@usage(MessageStreamEvent, Usage.output);
+  @@access(MessageStreamEvent, Access.internal);
+
+  @@usage(ErrorEvent, Usage.output);
+  @@access(ErrorEvent, Access.internal);
+
+  @@usage(DoneEvent, Usage.output);
+  @@access(DoneEvent, Access.internal);
+
+  // Ensure some shared types remain public despite interaction with previously internalized details
```

Comment:
can you remove `filename` from params as well for multipart formdata? https://github.com/Azure/azure-rest-api-specs/issues/28905

Reply:
Thank you for the link to the issue. We will address that in a separate PR. This PR is for Assistants V1 streaming support.